.github/workflows/ci.yml

```diff
@@ -53,10 +53,11 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest]
+        # We test on all LTS, maintenance and current versions of node above 22.X
```

Comment:
I think this is not true? Isn't 20.x LTS, but it's not included? We could try in a follow-up to add it, but i expect several things will fail.

Reply:
> I think this is not true?

it's said "above 22.X"

Reply:
Ah, it does say above 22 🤦‍♂️

* `tsx` fail dude to `findPackageJSON`
  * this we could gate (i think we did this for 22.x) and just continue excluding 20 from pjson.engines
* serval test fail dude to snapshot testing
  * ah, that's a show-stopper for supporting 20.x in CI.
* `pjson.test.mjs` fail dude to `globSync`
  * oh! That test should only be run once / in 1 version of node: it's not testing any integration with APIs. The only way it can fail across node versions is from a red-herring like globSync being missing.

Reply:
I think we can resolve this 
it simply reduces the annoying stuff.

And real question is "who use custimization hooks in v20" ??? 
not a lot of people